<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComposeWindow</class>
 <widget class="QMainWindow" name="ComposeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Mail</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TextRecipient">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>411</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>85</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Title:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="TextTitle">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>411</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="TextBody">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>741</width>
      <height>381</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonSend">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>520</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ButtonCancel">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>520</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
